tests/filecheck/dialects/riscv/riscv_ops.mlir

```diff
@@ -3,5 +3,7 @@
   %0 = "test.op"() : () -> !riscv.reg<>
   %1 = "test.op"() : () -> !riscv.reg<>
   %2 = "riscv.add"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
-// CHECK: %2 = "riscv.add"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  // CHECK: %2 = "riscv.add"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
```

Comment:
When I mentioned SSAValue naming as an issue, I was referring to testing any SSAValues names.

So in total, I think this should be `%{.*}` instead of `%3` (and all the others).

Reply:
Ah ok, I misunderstood, I'll fix this in the next PR